solc/CommandLineInterface.cpp

```diff
@@ -963,6 +970,12 @@ bool CommandLineInterface::processInput()
 			string validPath = _path;
 			if (validPath.find("file://") == 0)
 				validPath.erase(0, 7);
+			else if (validPath[0] == '.' ||
+					(validPath[0] == '/' && !boost::filesystem::exists(validPath)) ||
+					validPath[0] != '/')
+			{
+				validPath = m_basePath.string() + "/" + validPath;
+			}
```

Comment:
```suggestion
			validPath = m_basePath.string() + "/" + validPath;
```

but maybe use `boost::fs`'s `/` operator for appending.